src/components/molecules/UserUpdateForm.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { ChangeEvent, useState } from 'react';
+import DialogWithHeader from './DialogWithHeader';
+import { Box, Grid, TextField } from '@material-ui/core';
+import { useStore } from '../../store/storeConfig';
+import Button from '../atoms/Button';
+import { makeStyles } from '@material-ui/core/styles';
+import { observer } from 'mobx-react-lite';
+import { useTranslation } from 'react-i18next';
+
+interface IProps {
+  isShowing: boolean;
+  onClose: () => void;
+  defaultValues: IFormInput;
+}
+
+export interface IFormInput {
+  email?: string;
+  firstName?: string;
+  lastName?: string;
+  workplace?: string;
+}
+
```

Comment:
Just thoughts, if we have accept changing email after registration?

Reply:
theoretically yes, the API allows this.